ui/src/components/layouts/main-layout.tsx

```diff
@@ -1,12 +1,37 @@
 import { Outlet } from 'react-router'
+import cn from 'classnames'
+import { observer } from 'mobx-react-lite'
+import { useInjection } from 'inversify-react'
 
 import { Header } from '@/components/ui/header'
+import { Marquee } from '@/components/lib/marquee'
+import { ConditionalRender } from '@/components/lib/conditional-render'
 
-export const MainLayout = () => (
-  <>
-    <Header />
-    <main className='pageWrapper'>
-      <Outlet />
-    </main>
-  </>
-)
+import { CONTAINER_IDS } from '@/config/inversify/container-ids'
+
+import type { MarqueeVariant } from '@/components/lib/marquee'
+import type { AlertMessageLevel } from '@/types/alert-message-level.type'
+
+const ALERT_LEVEL_TO_MARQUEE_VARIANT_MAP: Record<AlertMessageLevel, MarqueeVariant> = {
+  ['Critical']: 'Error',
+  ['Information']: 'Info',
+  ['Warning']: 'Warn',
+}
```

Comment:
Not sure this belongs here. Maybe marquee can self contain everything related to the alert message?

Reply:
Maybe a better variant would be to create another component that would read the alert message settings and add a marquee if there is one